responsibleai/responsibleai/_managers/causal_manager.py

```diff
@@ -225,9 +225,13 @@ def compute(self):
             y = pd.concat([self._train, self._test], ignore_index=True)[
                 self._target_column].values.ravel()
 
+            categoricals = self._categorical_features
+            if categoricals is None:
+                categoricals = []
```

Comment:
Could you possibly add a comment saying that this is a limitation in econml and would be addressed by econml?

Reply:
Would add this in a follow up PR.